src/lib/drizzle-helpers.js

```diff
@@ -0,0 +1,48 @@
+import { sql } from 'drizzle-orm'
+import { assert } from '../utils.js'
+/** @import { BetterSQLite3Database } from 'drizzle-orm/better-sqlite3' */
+
+/**
+ * @param {unknown} queryResult
+ * @returns {number}
+ */
+const getNumberResult = (queryResult) => {
+  assert(
+    queryResult &&
+      typeof queryResult === 'object' &&
+      'result' in queryResult &&
+      typeof queryResult.result === 'number',
+    'expected query to return proper result'
+  )
+  return queryResult.result
+}
+
+/**
+ * Get the number of rows in a table using `SELECT COUNT(*)`.
+ * Returns 0 if the table doesn't exist.
+ *
+ * @param {BetterSQLite3Database} db
+ * @param {string} tableName
+ * @returns {number}
+ */
+export const tableCountIfExists = (db, tableName) =>
```

Comment:
```suggestion
export const countTableRows = (db, tableName) =>
```
Naming of this one is hard! Not sure if this is better. The original name suggested to me that it is counting tables ("tableCount"), and only counting if something exists ("IfExists"), where-as I think what this is doing is counting rows in a table whether it exists or not.

Reply:
`safeCountTableRows()`?

Reply:
Renamed it in f93b4a4dfbc45021e937003583c5d96c6c7edc0e. I also stopped exporting it in ae6e0c6d4d97b160f5336b79a578f80bc1aa7085.